filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -434,6 +434,10 @@ DECL_DRIVER_API_N(bindSamplers,
         uint32_t, index,
         backend::SamplerGroupHandle, sbh)
 
+DECL_DRIVER_API_N(writePushConstants,
+        backend::ShaderStage, stage,
+        backend::PushConstantArray, pushConstants)
```

Comment:
I would prefer an API like `setPushConstant(stage, index, value)` so that it's very efficient to set just a couple ones. Otherwise we'll have to send 128 bytes each time (possible per draw call) in the command stream.

Reply:
So should Material do the name to index mapping?

Reply:
done